lib/capability/capability.go

```diff
@@ -37,13 +37,13 @@ func (caps capabilitiesSort) Less(i, j int) bool { return string(caps[i]) < stri
 // SetCapabilities populates and returns cluster capabilities from ClusterVersion capabilities spec. This method also
 // ensures that no previously enabled capability is now disabled and returns any such implicitly enabled capabilities.
 func SetCapabilities(config *configv1.ClusterVersion,
-	existingEnabled, alwaysEnabled map[configv1.ClusterVersionCapability]struct{}) ClusterCapabilities {
+	existingEnabled []configv1.ClusterVersionCapability) ClusterCapabilities {
 
 	var capabilities ClusterCapabilities
-	capabilities.KnownCapabilities = setKnownCapabilities()
+	capabilities.KnownCapabilities = GetCapabilitiesAsMap(configv1.KnownClusterVersionCapabilities)
```

Comment:
This looks good, but I am a bit concerned about switching between two sources of truth for the same thing. 

- Previously, this was "all caps that were enabled in at least one capsets". Now this is a "all caps". So if there is a capability that is not enabled in any set, then we start to see it here but previously we did not. But that actually sounds correct - making this not a refactor but a fix?

- There's also `GetKnownCapabilities` in this file which could be replaced too?

Reply:
> making this not a refactor but a fix?

Described the logic change in the PR description.

Also removed `GetKnownCapabilities `.